### PR TITLE
Added TVShowTime Provider

### PR DESCRIPTION
**Pull request by Henokv**
This change is [Reviewable](https://reviewable.io/reviews/socialiteproviders/providers/65)

### CODE REVIEW

**Comment by faustbrian** [comment]
Why are some headers and tokens commented out?

**Comment by faustbrian** [comment]
@Henokv Why are some headers and tokens commented out?

**Comment by Henokv** [comment]
@faustbrian Sorry, they have two ways to authenticate. In query params and header. However I forgot to set it back to header authentication.

File: src/TVShowTime/Provider.php [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+            'id'       => $user['user']['id'],
+            'nickname' => $user['user']['name'],
+            'avatar'   => $user['user']['image'],
+        ]);
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    protected function getTokenFields($code)
```
Comment (faustbrian) [inline src/TVShowTime/Provider.php]: @Henokv Could you please remove this method as you are not using it for anything.

**Comment by faustbrian** [comment]
https://packagist.org/packages/socialiteproviders/tvshowtime

**Pull request merged**